bundles/org.openhab.binding.valloxmv/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -246,4 +258,110 @@
 		<category>Temperature</category>
 		<state pattern="%.2f %unit%" readOnly="false" />
 	</channel-type>
+
+	<channel-type id="boosttime">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Boost profile timer value</label>
+		<description>Boost profile timer value in minutes</description>
+		<state readOnly="false" min="1" max="65535" pattern="%d min" />
+	</channel-type>
+
+	<channel-type id="boosttimerenabled">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Boost profile timer enabled</label>
+		<description>Timer enabled setting in boost profile</description>
+		<state readOnly="false">
+			<options>
+				<option value="1">Enabled</option>
+				<option value="0">Disabled</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fireplaceextrfan" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fireplace extract fan speed</label>
+		<description>Fireplace profile extract fan speed in % (0-100)</description>
+		<state readOnly="false" min="0" max="100" pattern="%d %%" />
+	</channel-type>
+
+	<channel-type id="fireplacesuppfan" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fireplace supply fan speed</label>
+		<description>Fireplace profile supply fan speed in % (0-100)</description>
+		<state readOnly="false" min="0" max="100" pattern="%d %%" />
+	</channel-type>
+
+	<channel-type id="fireplacetime">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fireplace profile timer value</label>
+		<description>Fireplace profile timer value in minutes</description>
+		<state readOnly="false" min="1" max="65535" pattern="%d min" />
+	</channel-type>
+
+	<channel-type id="fireplacetimerenabled">
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
This sounds like a perfect match for `Switch`. Is there any special reason why you chose to implement it as `Number`? (also applies for the other enable/disable channels)

You could use `OnOffType.from(value == 1)` when reading and `state.as(DecimalType.class).intValue()` for writing.

Reply:
Agree, but it's just in line with what I did before (and that code shouldn't be changed to keep it stable for 2.5.x).

Reply:
Where exactly? I did not fine a single channel where a boolean value is modeled as a number (I'm not speaking of the multi-state channels)

Reply:
You are right. I did remember that incorrectly.
@anttiisoaho: Why isn't setting the status to fireplace/boost/etc. good enough here? The function is redundant to that, isn't it?

Reply:
I agree, those timerenabled channels (3) should be changed to Switches. The idea here is to have more control for profiles settings, one of the timer profiles setting is timer enabled/disabled. Status change function is ok if you have 15 minutes time for fireplace timer and 30 minutes time for boost timer, this is currently hardcoded into code. But if you want to have different times, then these times should be read from Vallox device. This specific topic is still WIP (I had added comment into code about this)

Reply:
Did you actually try wether it changes back after the timer number of minutes? IIRC from my Vallox it keeps the boost when I put it into boost mode forever ... (though it would be supposed to do so given the current code just for 30 min ...)

Reply:
Just tested it and the status change kept for longer than 30 min (likely forever). Maybe that is a bug in my Vallox MV 350 and so I didn't bother about that kind of missing implementation...
Thanks for correcting that.

Reply:
Just curious: do you have the similar behavior when changing to Fireplace profile?

Reply:
Now all timerenabled items are defined to switches. And now also proper timer values will be used when changing to boost or fireplace profile.